src/main/java/kafdrop/service/KafkaHighLevelAdminClient.java

```diff
@@ -105,6 +105,9 @@ ClusterDescription describeCluster() {
         configsByTopic.put(entry.getKey().name(), entry.getValue());
       }
     } catch (InterruptedException | ExecutionException e) {
+      if (e.getCause() instanceof UnsupportedVersionException) {
+        return new HashMap<>();
```

Comment:
Can you please use `Map.of()`. This is an empty map that does not require mutation.